obspy/io/nordic/core.py

```diff
@@ -898,13 +1162,17 @@ def blanksfile(wavefile, evtype, userid, overwrite=False, evtime=None):
         f.write(' ' + write_wavfile + '6'.rjust(79 - len(write_wavfile)) +
                 '\n')
         # Write final line of s-file
-        f.write(" STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU" +
-                " VELO AIN AR TRES W  DIS CAZ7\n")
+        if version == 'OLD':
```

Comment:
It might be worth defining these outside these functions (here and `write_select`) to avoid possible typos?

Reply:
good point; also to avoid the duplication. done!